src/config/config.types.ts

```diff
@@ -1,19 +1,16 @@
+import { AvoOrHetArchief } from '@meemoo/admin-core-api';
```

Comment:
Really need to move away from these naming schemes, they're not clear nor scalable when additional applications are added.

Reply:
agreed, in the future we might want to go towards: applicationName: 'avo' | 'hetarchief' | '...'
for now this name is clear, but not scalable.